templates/observation.html

```diff
@@ -851,6 +846,63 @@ <h5 class="modal-title" id="confirmModalLabel">Reset media</h5>
         selection: false
       });
 
+      const canvasHeight = 332;
+      const canvasWidth = 530;
+
+      if (!is_touch_device) {
+        stage.on('mouse:wheel', function(opt) {
+          var delta = opt.e.deltaY * -1;
```

Comment:
What is e? Is this event? It is not clear. Could you assign `opt.e` to variable (called e.g. event)?